app/models/name.rb

```diff
@@ -146,9 +146,59 @@
 #  alt_ranks::               Ranks: map alternatives to our values.
 #
 #  ==== Scopes
+#  created_on("yyyymmdd")
+#  created_after("yyyymmdd")
+#  created_before("yyyymmdd")
+#  created_between(start, end)
+#  updated_on("yyyymmdd")
+#  updated_after("yyyymmdd")
+#  updated_before("yyyymmdd")
+#  updated_between(start, end)
+#  of_lichens
+#  not_lichens
+#  deprecated
+#  not_deprecated
+#  with_description
+#  without_description
+#  description_includes
+# TODO: Add with_descriptions_by_author https://github.com/MushroomObserver/mushroom-observer/pull/1082/files?show-viewed-files=true&file-filters%5B%5D=#r928141093
```

Comment:
@nimmolo: I did not do this because -- based on its name -- 
`with_descriptions_by_author` duplicates
`with_descriptions_created_by`.
If you mean something different, please let me know.
If not, maybe we should just create an alias.